graphql_compiler/cost_estimation/statistics.py

```diff
@@ -82,6 +82,35 @@ def get_distinct_field_values_count(self, vertex_name, field_name):
         """
         return None
 
+    def get_field_quantile(self, vertex_name, field_name, fraction):
+        """Return the field value that's larger than a given fraction of all values.
+
+        Args:
+            vertex_name: str, name of a vertex defined in the GraphQL schema.
+            field_name: str, name of a vertex field.
+            fraction: float from 0 to 1, the fraction of values smaller than the result
```

Comment:
nit: remove, "the fraction of values smaller than the result". It does not add much information and the result may be None so it is technically not true.